src/main/java/com/github/maxopoly/WurstCivTools/ConfigParser.java

```diff
@@ -57,6 +54,10 @@ public void parseTools(ConfigurationSection config) {
 			WurstEffect effect;
 			switch (type) {
 			case "PYLONFINDER":
+				if (current.getBoolean("enabled",false) == false){
```

Comment:
I thought it would be nice to have explicit feature disabling/enabling without having to butcher the config to disable/enable features.